docs/authenticated-pages.md

```diff
@@ -0,0 +1,33 @@
+# Running Lighthouse on Authenticated Pages
+
+Default runs of Lighthouse load a page as a "new user", with no previous session or storage data. This means that pages requiring authenticated access do not work without additional setup.
+
+## Puppeteer
+
+[Puppeteer](https://pptr.dev) is the most flexible approach for auditing pages behind authentication with Lighthouse. See [recipes/auth](./recipes/auth) for more.
```

Comment:
```suggestion
[Puppeteer](https://pptr.dev) is the most flexible approach for auditing pages behind authentication with Lighthouse. 

See [a working demo at /docs/recipes/auth](./recipes/auth).
```

since there's so much good stuff at the recipe link, I want to make it bigger and more obvious. :)